pkg/runtime/plan/union.go

```diff
@@ -34,6 +36,8 @@ import (
 	"github.com/arana-db/arana/pkg/util/log"
 )
 
+var Tracer = otel.Tracer("ExecPlan")
```

Comment:
How about moving it into a standalone file? For example, `pkg/runtime/plan/trace.go`, because it doesn't belong to `union.go` logically.